docker/nodejs-build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos/nodejs-8-centos7
+FROM centos/nodejs-10-centos7
```

Comment:
@WadeBarnes we need to update the service build for OpenShift as well, otherwise this will fail. See [here](https://github.com/bcgov/orgbook-configurations/blob/master/openshift/templates/frontend/frontend-artifacts-build.json#L138-L158). Probably good to hold-off from merging until we have updated that.

Reply:
@amanji I am updating the build, will post back shortly once I have something that will work with Angular 11

Reply:
This should be good to go now, the build has been updated to use Node 10